.rubocop.yml

```diff
@@ -61,11 +60,8 @@ Style/NoFetchWithLiteralNilDefault:
 Style/UseGovukLinkHelper:
   Enable: true
 
-# past migrations are ignored; we don't expect to go back and edit them ever,
-# we'll fix them in future migrations if at all.
 Rails/BulkChangeTable:
-  Exclude:
-    - db/migrate/202[0123]*
+  Enabled: false
```

Comment:
Can we just disable this in the specific individual migrations rather than for everything?